pkg/controller/stack/stack_controller.go

```diff
@@ -1229,14 +1235,15 @@ contexts:
 - context:
     cluster: local
     user: local
+    namespace: %s
```

Comment:
Should we add an environment variable to potentially disable this? Agree with this being the default though.

Reply:
How about `PULUMI_INFER_NAMESPACE` ?